app/services/queue_service.rb

```diff
@@ -36,10 +36,10 @@ def step_name
   # Generate the queue name from step
```

Comment:
should "from step" be removed in this PR?

Reply:
it is still being derived from the current step object.